_posts/2021-02-26-feather-nrf52840.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: blog
+title: How to Port a Board for RIOT
+authors: Martine S. Lenders
```

Comment:
BTW, this isn't shown (and I am not sure I am using the right meta info here... just copied what I had in Pelican). IMHO if we want blog posts by external authors, some attribution would be nice ;-).

Reply:
See #21